runtime/compiler/CMakeLists.txt

```diff
@@ -346,6 +353,17 @@ if(JITSERVER_SUPPORT)
 	target_link_libraries(j9jit PRIVATE ${Protobuf_LIBRARIES})
 endif()
 
+# This is a bit hokey, but cmake can't track the fact that files are generated across directories.
+# Note: while these are only needed on z, setting the properties unconditionally has no ill-effect.
+set_source_files_properties(
+	${CMAKE_CURRENT_BINARY_DIR}/z/runtime/Math.s
```

Comment:
Are these needed both here and in z/runtime/CMakeLists.txt?

Reply:
I think this only sets the properties, while the one in z/runtime/CMakeLists.txt actually generates them.

Reply:
Yes. this just tells cmake that the files are generated elsewhere (otherwise it will error out saying that the source files do not exist)